src/agent/vendor.hpp

```diff
@@ -36,9 +36,12 @@
 
 #include "openthread-br/config.h"
 
-#include "ncp/ncp_openthread.hpp"
+#include "agent/application.hpp"
 
 namespace otbr {
+
+class Application;
```

Comment:
Is the forward declaration needed? You've already included the `application.hpp`.

Reply:
I used this to fix compile issue.